atd-vzd/metadata/databases/default/tables/public_atd_txdot_crashes.yaml

```diff
@@ -1715,3 +1715,16 @@ update_permissions:
       filter: {}
       check: null
     comment: ""
+delete_permissions:
+  - role: editor
+    permission:
+      filter:
+        temp_record:
+          _eq: true
+    comment: "Allow for the deletion of temp records only"
+  - role: vz-admin
+    permission:
+      filter:
+        temp_record:
+          _eq: true
+    comment: "Allow for the deletion of temp records only"
```

Comment:
this is a nice improvement, by the way!

Reply:
+1 - now i know the power of the filter here 💪 ✨